src/components/Tag/Tag.stories.tsx

```diff
@@ -0,0 +1,10 @@
+import Tag from "./Tag";
+
+export default {
+  title: "Component/Tag",
+  component: Tag,
+};
+
+export const Today = (): JSX.Element => <Tag tagName="heute" value="today" />;
+export const Monday = (): JSX.Element => <Tag tagName="Mo" value="day1" />;
+export const Tuesday = (): JSX.Element => <Tag tagName="Di" value="day2" />;
```

Comment:
great Idear to make the Tags as a radio inputs. Love it.

In storybook I see that the color chnages just one time to red. from the code I get, it is red(primary) when checked, and not red(seconday color) when unchecked. Maybe there is some issue, maybe it is just storybook.

Reply:
Great, thank you! :)
However, that is a normal behavior for a radio button. You can usually not "uncheck". I will consider implementing this later!